.gitignore

```diff
@@ -1,4 +1,5 @@
 # dependencies
+/AutomationDemo/node_modules
```

Comment:
this is redundant, next line covers it

Reply:
Also - should we not rename the folder from 'automationDemo' to 'tests' or something? Although this first example's purpose was to be demoed, I'd hope this to be the first of a full suite.

Reply:
Renamed the directory to `AutomationTests`, but it did not seem like just `/node_modules` covered also the automation project node modules, as when I tried to remove that line, node modules were included to changes made.

Reply:
The patterns from .gitignore file are applied recursively to subdirectories as well.

We usually call directories with a postfix slash, like `node_modules/` so it would match even if used as a relative path (although git always uses absolute paths when matching).